specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-02-01/management.JSON

```diff
@@ -0,0 +1,2000 @@
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The fully qualified ID for the parent management group.  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000"
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the parent management group"
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The friendly name of the parent management group."
+        }
+      }
+    },
+    "ManagementGroupChildType": {
+      "type": "string",
+      "enum": [
+        "Microsoft.Management/managementGroups",
+        "/subscriptions"
+      ],
+      "description": "The type of child resource."
+    },
+    "OperationResults": {
+      "description": "The results of an asynchronous operation.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The fully qualified ID for the management group.  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of the resource.  For example, Microsoft.Management/managementGroups",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the management group. For example, 00000000-0000-0000-0000-000000000000",
+          "readOnly": true
+        },
+        "properties": {
+          "title": "Properties",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ManagementGroupInfoProperties"
+        }
+      }
+    },
+    "AzureAsyncOperationResults": {
+      "description": "The results of Azure-AsyncOperation.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The fully qualified ID for the management group.  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of the resource.  For example, Microsoft.Management/managementGroups",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the management group. For example, 00000000-0000-0000-0000-000000000000",
+          "readOnly": true
+        },
+        "status": {
+          "type": "string",
+          "description": "The current status of the asynchronous operation performed . For example, Running, Succeeded, Failed",
+          "readOnly": true
+        },
+        "properties": {
+          "title": "Properties",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ManagementGroupInfoProperties"
+        }
+      }
+    },
+    "HierarchySettings": {
```

Comment:
You should reference the common types to bring in 'id', 'name', 'type'...

https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v1/types.json